site/documentation/content/api/tenant/index.md

```diff
@@ -105,16 +105,17 @@ It is an error to include payload that is not of this type.
+| *resource-limits*      | *no*      | *object*  | Any resource limits that should be enforced for the tenant, e.g. the maximum number of concurrent connections and the maximum data volume for a given period. Refer to [Resource Limits Configuration Format]({{< relref "#resource-limits-configuration-format" >}}) for details.                                                                                                                                                                                                                           |
+| *tenant-id*            | *yes*     | *string*  | The ID of the tenant.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| *tracing*              | *no*      | *object*  | A set of options regarding the tracing of messages for the tenant. See [Tracing Format]({{< relref "#tracing-format" >}}) for a definition of the content model of the object.                                                                                                                                                                                                                                                                                                                               |
+| *trusted-ca*           | *no*      | *array*   | The list of trusted certificate authorities to use for validating certificates presented by devices of the tenant for authentication purposes. See [Trusted Certificate Authority Format]({{< relref "#trusted-ca-format" >}}) for a definition of the content model of the objects contained in the array. **NB** If the element is provided then the list MUST NOT be empty.                                                                                                                               |
+| *ext*                  | *no*      | *object*  | Arbitrary *extension* properties belonging to the tenant. See [Extension properties Format]({{< relref "#extension-properties-format" >}}) for details.                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
Could you please revert the above formatting changes and make use of the existing formatting style.